<launch>
    <rosparam file="$(find aleph2_hardware_interface)/config/hardware.yaml" command="load"/>
    <rosparam file="$(find aleph2_hardware_interface)/config/controllers.yaml" command="load"/>
    <rosparam file="$(find aleph2_hardware_interface)/config/joint_limits.yaml" command="load"/>
    <node name="controller_manager" pkg="aleph2_hardware_interface" type="controller_manager" output="screen"/>
    <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen"
        args="
            /aleph2/controller/state
            /aleph2/controller/velocity/gimbal1_pan
        "/>
</launch>